index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header class="toolbar">
+        <a class="logo">My Logo</a>
```

Comment:
@vickymarz is ti exactly as the example? talking about this part ' href="/" ' as i havent used it before

Reply:
@agneta1991, you can use whatever you want in as much as the path is exactly as the homepage. 

For me not to impose on you, you can still use it exactly as the example because it works as well. The forward slash will link to your homepage as default